pages/_app.tsx

```diff
@@ -21,11 +22,13 @@ export default function MyApp({ Component, pageProps }: MyAppProps) {
     // Todo: get rid of the StoreProvider. (Simplify our workflow by using React-Query for server state data-handling and interactions)
     <StoreProvider store={store}>
       <QueryClientProvider client={queryClient}>
-        <MsalProvider instance={msalInstance}>
-          <LayoutWrapper {...pageProps}>
-            <Component {...pageProps} />
-          </LayoutWrapper>
-        </MsalProvider>
+        <SelectedNodeForQIPRProvider>
+          <MsalProvider instance={msalInstance}>
+            <LayoutWrapper {...pageProps}>
+              <Component {...pageProps} />
+            </LayoutWrapper>
+          </MsalProvider>
+        </SelectedNodeForQIPRProvider>
```

Comment:
Can we instead add this to the FlowWrapper inside Flow.tsx? Then the provider can be reused in different flows in the future.

Reply:
Fixed